src/components/Book.jsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
```

Comment:
- [ ]  As requested by the previous reviewer, Kindly ensure that you did not disable any linter, Remember that linters are there as a guide to check issues in your code. :pray:

Reply:
I tried to follow the instructions from the documentation but nothing is changing.
Please consider my request 🙏 🙏 🙏

Reply:
@Fombi-Favour I suggest that you find a way to fix it. Read more about `import/no-extraneous-dependencies` here :point_right: https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-extraneous-dependencies.md

Reply:
I read the article and use the command recommended and nothing changed. the same issue.
This is not the first time it has happen to me
Please consider my request 🙏 🙏 🙏